### PR TITLE
integrate dev into master

### PR DESCRIPTION
**Pull request by MiguelAlho**
Hoping we could move  this stuff into master to have a 4.1 or a 4.0.1

### CODE REVIEW

**Comment by nblumhardt** [comment]
Hi @MiguelAlho - the version published when this reaches master will be determined here:

https://github.com/serilog/serilog-sinks-loggly/blob/dev/src/Serilog.Sinks.Loggly/project.json#L2

I.e. if the next master package should be 4.0.1, then we need to update the version on the dev branch first. This will publish dev packages with that version and a -dev suffix, and publish the un-suffixed package when it hits master.

To summarize, the PR needs to bring the desired version with it :-)

**Comment by MiguelAlho** [comment]
Thanks! I wasn't sure if this was automatic or not (i.e. integrated into the build process). I've added the commit to bump the version to 4.0.1
